### PR TITLE
Export reserved word

### PR DESCRIPTION
**Pull request by roopakv**
When using jennifer we realized we started making our own list of reserved keywords. It sort of beocmes inevitable. Thought it might make sense to export this so that we can use it too.

LMK if you think we shouldn't do this

### CODE REVIEW

File: jen/reserved.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package jen
+
+var reserved = []string{
+	/* keywords */
+	"break", "default", "func", "interface", "select", "case", "defer", "go", "map", "struct", "chan", "else", "goto", "package", "switch", "const", "fallthrough", "if", "range", "type", "continue", "for", "import", "return", "var",
+	/* predeclared */
+	"bool", "byte", "complex64", "complex128", "error", "float32", "float64", "int", "int8", "int16", "int32", "int64", "rune", "string", "uint", "uint8", "uint16", "uint32", "uint64", "uintptr", "true", "false", "iota", "nil", "append", "cap", "close", "complex", "copy", "delete", "imag", "len", "make", "new", "panic", "print", "println", "real", "recover",
+	/* common variables */
```
Comment (roopakv) [inline jen/reserved.go]:
```suggestion
  /* common variables */
```

File: jen/reserved.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package jen
+
+var reserved = []string{
+	/* keywords */
+	"break", "default", "func", "interface", "select", "case", "defer", "go", "map", "struct", "chan", "else", "goto", "package", "switch", "const", "fallthrough", "if", "range", "type", "continue", "for", "import", "return", "var",
+	/* predeclared */
+	"bool", "byte", "complex64", "complex128", "error", "float32", "float64", "int", "int8", "int16", "int32", "int64", "rune", "string", "uint", "uint8", "uint16", "uint32", "uint64", "uintptr", "true", "false", "iota", "nil", "append", "cap", "close", "complex", "copy", "delete", "imag", "len", "make", "new", "panic", "print", "println", "real", "recover",
+  /* common variables */
```
Comment (roopakv) [inline jen/reserved.go]:
```suggestion
   /* common variables */
```

**Comment by nirajjayantbolt** [comment]
@dave any updates on this PR?

**Comment by dave** [comment]
Done. Tagged as v1.4.0.

**Pull request merged**